components/lib/autocomplete/AutoCompleteBase.js

```diff
@@ -86,7 +86,7 @@ const styles = `
         list-style-type: none;
     }
 
-    .selected {
+    .p-highlight {
         background-color: rgba(6, 182, 212, 0.24) !important;
```

Comment:
Should it be `!important` ?

Reply:
Removed the style added for p-highlight as a default style was already applied for autocomplete p-highlight class, and its working as expected.